test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AttributeRoutingTest.cs

```diff
@@ -40,20 +42,19 @@ public class AttributeRoutingTest
                 "Error: The route parameter name 'a/dkfk' is invalid. Route parameter names must be non-empty and " +
                 "cannot contain these characters: '{', '}', '/'. The '?' character marks a parameter as optional, " +
                 "and can occur only at the end of the parameter. The '*' character marks a parameter as catch-all, " +
-                "and can occur only at the start of the parameter." + Environment.NewLine +
-                "Parameter name: routeTemplate";
-            
+                "and can occur only at the start of the parameter.";
+
             var services = CreateServices(action);
 
             var route = AttributeRouting.CreateAttributeMegaRoute(services);
 
             // Act & Assert
             var ex = await Assert.ThrowsAsync<InvalidOperationException>(async () =>
-            {
-                await route.RouteAsync(new RouteContext(new DefaultHttpContext()));
-            });
+                {
+                    await route.RouteAsync(new RouteContext(new DefaultHttpContext()));
+                });
 
-            Assert.Equal(expectedMessage, ex.Message);
+            Assert.Contains(expectedMessage, ex.Message);
```

Comment:
This test will fail once our product is localized. Use `[ReplaceCulture]` and stick w/ the previous `expectedMessage`.